example_internal_load_balancer.py

```diff
@@ -0,0 +1,605 @@
+        )
+    ]
+    lb_ob_rule = network_client.load_balancers.begin_create_or_update(
+        GROUP_NAME,
+        LB_NAME,
+        load_balancer_parameters
+    ).result().outbound_rules
+    print("Create Outbound Rule for the Outbound Backend Pool:\n{}".format(lb_ob_rule[-1]))
+
+    # Add VMs to the Outbound Pool
+    ob_pool_info = network_client.load_balancer_backend_address_pools.get(
+        GROUP_NAME,
+        LB_NAME,
+        OB_POOL_NAME
+    )
+    for i in range(3):
+        vmnic_info = network_client.network_interfaces.get(
+            GROUP_NAME, NIC_NAMES[i]
+        )
+        for ip_conf in vmnic_info.ip_configurations:
+            if ip_conf.name == NIC_IP_CONFIG_NAME:
+                ip_conf.load_balancer_backend_address_pools.append(ob_pool_info)
+
+        add_vm_to_lb_op = network_client.network_interfaces.begin_create_or_update(
+            GROUP_NAME,
+            NIC_NAMES[i],
+            vmnic_info
+        ).result().ip_configurations
+        print("Add {} to Load Balancer Outbound Pool: \n{}".format(NIC_NAMES[i], add_vm_to_lb_op))
+
+    # Get the Public IP Address of the Load Balancer
+    network_public_ip_info = network_client.public_ip_addresses.get(
+        GROUP_NAME, PUBLIC_IP_NAME
+    )
+    print("Public IP Address:\n{}".format(network_public_ip_info.ip_address))
+
+
+
+
+def delete_resource_group():
+    subscription_id = os.environ.get(
+        'AZURE_SUBSCRIPTION_ID',
+        '11111111-1111-1111-1111-111111111111')  # your Azure Subscription Id
+
+    resource_client = ResourceManagementClient(
+        credential=DefaultAzureCredential(),
+        subscription_id=subscription_id
+    )
+    print('Delete Resource Group')
+    resource_client.resource_groups.begin_delete(
+        GROUP_NAME)
+
+    print('Success')
+
+
+
+def run_install_iis():
+    subscription_id = os.environ.get(
+        'AZURE_SUBSCRIPTION_ID',
+        '11111111-1111-1111-1111-111111111111')  # your Azure Subscription Id
+
+    compute_client = ComputeManagementClient(
+        credential=DefaultAzureCredential(),
+        subscription_id=subscription_id
+    )
+    RG_NAME = 'CreatePubLBQS-rg'
+    VM2_NAME = 'myVM2'
+    VM_EXT_NAME = 'CustomScriptExtension'
+    SETTINGS = '{"commandToExecute":"powershell Add-WindowsFeature Web-Server; powershell Add-Content -Path \"C:\\inetpub\\wwwroot\\Default.htm\" -Value $("Hello World from "+$env:computername)"}'
+    vm_ext_parameters = compute_models.VirtualMachineExtension(
+        location='eastus',
+        publisher='Microsoft.Compute',
+        type_handler_version='1.8',
+        settings=SETTINGS,
+    )
+    vm_ext = compute_client.virtual_machine_extensions.begin_create_or_update(
+        RG_NAME, VM2_NAME, VM_EXT_NAME, vm_ext_parameters).result()
+
+    print("Add vm_ext:\n{}".format(vm_ext))
+
+
+if __name__ == "__main__":
+    run_example()
+    # run_install_iis()
+    # delete_resource_group()
```

Comment:
add annotation like `#  run it if you want to delete all the resource created before`